tests/test_modules.py

```diff
@@ -101,3 +102,15 @@ def func(x):
         for f in info['pkg_ver_list']:
             print(f[0])
 
+class InteractiveTest(unittest.TestCase):
```

Comment:
Can we add a little bit of documentation here on what we're trying to test, and explain the difference between interactive vs regular mode?